.github/actions/vtk_commit_sha

```diff
@@ -1 +1 @@
-ea2b6b5ad1516f65684eff0972a198af873ca1c1
+7ae47ffa119c6feaea43ef021fb48a3056abbec2
```

Comment:
superbuild update needed

Reply:
Is it still needed now? You changed the SB to retrieve the sha from the master branch, right?

Reply:
Yes indeed, not needed anymore, the superbuild will grab it for the next nightly after you merge. Keep an eye on the nightly run results btw.